src/utils/helpers/bcrypt/hash.js

```diff
@@ -0,0 +1,16 @@
+const bcrypt = require('bcrypt');
+
+const hashPassword = (password) => bcrypt.hash(password, 10);
+
+const logHash = async () => {
+  try {
+    const hash = await hashPassword('Admin@codeacademy@13');
+    console.log(hash);
+    return hash;
+  } catch (err) {
+    return console.log(err);
+  }
+};
+
+logHash();
+// export default hashPassword;
```

Comment:
I think you forget to delete this file after you finish generating the password to fake data, so delete it